### DISCUSSION TITLE
Adding multiple Github users

### DISCUSSION THREAD

**Discussion by PaulT77** [community/community#137631] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, Can we add or import multiple users and send requests to join a GitHub organization? If that is possible how can we do it?

**Comment by ThisIs-Developer** [comment]
### Inviting a User to Your GitHub Organization

1. **Go to Organizations:** Click your profile photo, then select "Your organizations."
2. **Select Organization:** Choose your organization’s name.
3. **Access People:** Click "People" under the organization name.
4. **Invite Member:** Click "Invite member."
5. **Enter Details:** Type the username or email, set their role, and optionally add to a team.
6. **Send Invitation:** Click "Send invitation."

The user will receive an email to accept the invitation.

### If this is helpful, please mark as answer! 😴
## For more details, visit the [GitHub documentation](https://docs.github.com/en/organizations/managing-membership-in-your-organization/inviting-users-to-join-your-organization).
![answer](https://github.com/user-attachments/assets/98626be6-a7a7-4a5b-831c-1efc27de9911)

**Comment by greenmmq** [comment]
If you are on Windows and have `gh` installed, you can use this powershell script:

emails.txt:
```shell
user1@company.com
user3@company.com
user2@company.com
...
```

```powershell
# Path to the file containing the email addresses
$emailFilePath = "C:\path\to\emails.txt"

# Define your GitHub organization name
$orgName = "your-org-name"

# Read all emails from the file
$emails = Get-Content $emailFilePath

# Loop through each email and send the invitation
foreach ($email in $emails) {
    Write-Output "Inviting $email to $orgName..."

    $result = gh api `
        --method POST `
        -H "Accept: application/vnd.github.v3+json" `
        "/orgs/$orgName/invitations" `
        -f email="$email" `
        -f role="direct_member"

    # Check if the command succeeded
    if ($?) {
        Write-Output "Successfully invited: $email"
    } else {
        Write-Output "Failed to invite: $email"
    }
}
```
